src/getMetric.js

```diff
@@ -0,0 +1,384 @@
+* @param {date} to The ending date to fetch the data. Example: DATE(2018, 9, 21)
+* @param {string} timeBound The metric is calculated only by taking into account the
+* tokens and coins that have moved in the past number of years or days.
+* @returns {number} returns the daily low price.
+* @customfunction
+*/
+function SAN_DAILY_LOW_PRICE_USD (projectSlug, from, to, timeBound) {
+  return handleErrors_(getMetric_)('daily_low_price_usd', projectSlug, from, to, timeBound)
+}
+
+/**
+* Gets the the daily opening price
+* @param {string} projectSlug Name of the asset at sanbase,
+* which can be found at the end of the URL (eg. the Santiment URL is
+* https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).
+* @param {date} from The starting date to fetch the data. Example: DATE(2018, 9, 20)
+* @param {date} to The ending date to fetch the data. Example: DATE(2018, 9, 21)
+* @param {string} timeBound The metric is calculated only by taking into account the
+* tokens and coins that have moved in the past number of years or days.
+* @returns {number} returns the daily opening price.
+* @customfunction
+*/
+function SAN_DAILY_OPENING_PRICE_USD (projectSlug, from, to, timeBound) {
+  return handleErrors_(getMetric_)('daily_opening_price_usd', projectSlug, from, to, timeBound)
+}
+
+/**
+* Gets the the daily trading volume
+* @param {string} projectSlug Name of the asset at sanbase,
+* which can be found at the end of the URL (eg. the Santiment URL is
+* https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).
+* @param {date} from The starting date to fetch the data. Example: DATE(2018, 9, 20)
+* @param {date} to The ending date to fetch the data. Example: DATE(2018, 9, 21)
+* @param {string} timeBound The metric is calculated only by taking into account the
+* tokens and coins that have moved in the past number of years or days.
+* @returns {number} returns the daily trading volume.
+* @customfunction
+*/
+function SAN_DAILY_TRADING_VOLUME_USD (projectSlug, from, to, timeBound) {
+  return handleErrors_(getMetric_)('daily_trading_volume_usd', projectSlug, from, to, timeBound)
+}
+
+/**
+* Gets the the daily active addresses
+* @param {string} projectSlug Name of the asset at sanbase,
+* which can be found at the end of the URL (eg. the Santiment URL is
+* https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).
+* @param {date} from The starting date to fetch the data. Example: DATE(2018, 9, 20)
+* @param {date} to The ending date to fetch the data. Example: DATE(2018, 9, 21)
+* @param {string} timeBound The metric is calculated only by taking into account the
+* tokens and coins that have moved in the past number of years or days.
+* @returns {number} returns the daily active addresses.
+* @customfunction
+*/
+function SAN_DAILY_ACTIVE_ADDRESSES (projectSlug, from, to, timeBound) {
```

Comment:
@IvanIvanoff  Should we have `DAILY` in the function name? What is the chance to have the period as an argument in the future?